src/installer-win/installer.proj

```diff
@@ -0,0 +1,22 @@
+<!--
+  This project pulls down the setup compiler from a nuget and invokes it to build the setup exe.
+  It depends on the published self-contained bicep.exe and bicep.pdb to be placed in a bicep subfolder.
+  During the CI build, this happens via artifact download action.
+-->
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Does this mean the machine has to have .NET 4.6 installed to run the installer?

Reply:
Nope, it's just a dummy target framework needed for a NoTargets project. The installer is not .net-based btw. I believe it's native code (Delphi I think.)